### ISSUE TITLE
Add backend callbacks for authenticated data channels

### ISSUE THREAD

**Issue by AlgoQ** [bmoscon/cryptofeed#847]
Would it be possible to add backend callbacks for the authenticated data channels (in particular Redis)?

**Comment by bmoscon** [comment]
@AlgoQ yep, shouldnt be too difficult. thanks for the ask - wasnt sure if it would be used or not, so thanks for confirming
